src/node_config.cc

```diff
@@ -95,6 +95,14 @@ static void Initialize(Local<Object> target,
     }
   }
 
+  if (!env->options()->experimental_policy.empty()) {
```

Comment:
you can pull this value using `getOptions('--experimental-policy')`

Reply:
yes, please don’t add new properties to `process.binding('config')`

Reply:
where should it go

Reply:
@bmeck What @devsnek said – you can use `getOptions('--experimental-policy')` from `internalBinding('options')` for this

Reply:
can the new parser not differentiate

```
./node --experimental-policy ""
```

and

```
./node
```

?

Reply:
@bmeck I mean, neither did the old one (and the current state of the PR doesn’t disambiguate between them either)

You can look for e.g. `has_eval_string` in `node_options.cc` for an example case where we need to differentiate between a string option being set or not set

Reply:
previous parser allowed testing for typeof string to work since it only added when it was encountered, so i got confused for a sec when my code testing typeof from JS stopped working

Reply:
fixed, rebased